src/plugins/validation/oas3/semantic-validators/responses.js

```diff
@@ -75,6 +75,13 @@ module.exports.validate = function({ resolvedSpec }, config) {
           );
         } else {
           for (const statusCode of successCodes) {
+            if (!obj[statusCode].description) {
+              messages.addMessage(
+                path.concat([statusCode]),
+                `A ${statusCode} response should include a description.`,
+                'error'
+              );
+            }
```

Comment:
This looks good. I actually think we should move this check up to line 55 since this is required of all responses, not just success responses.